File: shoring/blackjackEngine/src/main/java/com/panopset/blackjackEngine/BlackjackMessagesDft.java
package com.panopset.blackjackEngine;

public class BlackjackMessagesDft implements BlackjackMessages {

  @Override
  public String getWonMsg() {
    return "Won";
  }

  @Override
  public String getLostMsg() {
    return "Lost";
  }

  @Override
  public String getPushMsg() {
    return "Push";
  }

  @Override
  public String getEvenMsg() {
    return "Even money";
  }

  @Override
  public String getBlackjackMsg() {
    return "Blackjack";
  }

  @Override
  public String getBustedMsg() {
    return "Busted";
  }

  @Override
  public String getDoubleImpossibleMsg() {
    return "Double not possible here";
  }

  @Override
  public String getSurrenderNotAllowedMsg() {
    return "Surrender not allowed in this casino";
  }

  @Override
  public String getSurrenderImpossibleMsg() {
    return "Surrender not possible here";
  }

  @Override
  public String getHandActiveMsg() {
    return "Hand is still active";
  }

  @Override
  public String getShuffledMsg() {
    return "Shuffled";
  }

  @Override
  public String getPleaseSelectMsg() {
    return "Please select";
  }

  @Override
  public String getDealMsg() {
    return "Deal";
  }

  @Override
  public String getResetMsg() {
    return "Reset";
  }
}

File: code/shoring/blackjackEngine/src/main/kotlin/com/panopset/blackjackEngine/BlackjackMessagesDft.kt
package com.panopset.blackjackEngine

class BlackjackMessagesDft : BlackjackMessages {
    override val wonMsg: String
        get() = "Won"
    override val lostMsg: String
        get() = "Lost"
    override val pushMsg: String
        get() = "Push"
    override val evenMsg: String
        get() = "Even money"
    override val blackjackMsg: String
        get() = "Blackjack"
    override val bustedMsg: String
        get() = "Busted"
    override val doubleImpossibleMsg: String
        get() = "Double not possible here"
    override val surrenderNotAllowedMsg: String
        get() = "Surrender not allowed in this casino"
    override val surrenderImpossibleMsg: String
        get() = "Surrender not possible here"
    override val handActiveMsg: String
        get() = "Hand is still active"
    override val shuffledMsg: String
        get() = "Shuffled"
    override val pleaseSelectMsg: String
        get() = "Please select"
    override val dealMsg: String
        get() = "Deal"
    override val resetMsg: String
        get() = "Reset"
}
